t/udp-generator.pl

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/perl
+
+# Read datagrams from input files and send them to a specified server
+
+use strict;
```

Comment:
```suggestion
use strict;
use warnings;
```

Generally speaking it's good to enable both strict and warning.